engine/src/main/java/org/terasology/rendering/nui/widgets/models/JsonTreeNode.java

```diff
@@ -21,8 +21,20 @@
  * The value type for the tree representation of a {@link JsonElement}.
  */
 public class JsonTreeNode {
+    /**
+     * The type of the JSON node.
+     */
     public enum ElementType {
-        PRIMITIVE, ARRAY, OBJECT, NULL
+        // Primitive data type (string, boolean, array).
```

Comment:
you oculd use a javadoc doc here. It is also possible as one liner if you wanted that e.g.`/** Some javadoc */`